omnibus-base-mainnet-andromeda.toml

```
@@ -220,7 +247,7 @@ defaultValue = "synthetix:3.3.18"
 defaultValue = "op-gas-price-oracle:3.4.0"
 
 [setting.spot_market_package]
-defaultValue = "synthetix-spot-market:3.3.15"
+defaultValue = "synthetix-spot-market:3.6.0"
```

should we revert this change?
```suggestion
defaultValue = "synthetix-spot-market:3.3.15"
```

---

helllo sir, db made changes recently to the latest versions and we'll go live to mainnet with those changes

---

No problem. I saw those versions reverted to old ones on testnet and wanted to double-check.